features/security/pom.xml

```diff
@@ -351,6 +351,11 @@
             <artifactId>security-idp-plugin-audiencerestriction</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
```

Comment:
❓ If you are only adding a bundle to a feature file, does the module containing the feature file need the bundle as a dependency? I don't know the answer, but this could have been the reason I was having errors in my feature loading for another ticket.

Reply:
This is late, but the answer to your question is "yes" btw 😄